mw-webapp/src/dataAccessLogic/GoalPreviewDAL.ts

```diff
@@ -0,0 +1,20 @@
+import {goalDTOToGoalPreviewConverter} from "src/dataAccessLogic/DTOToPreviewConverter/goalDTOToGoalPreviewConverter";
+import {GoalPreview} from "src/model/businessModelPreview/GoalPreview";
+import {GoalService} from "src/service/GoalService";
+
+/**
+ * Provides methods to interact with the GoalPreview model
+ */
+export class GoalPreviewDAL {
+
+  /**
+   * Get GoalsPreview
+   */
+  public static async getGoalsPreview(): Promise<GoalPreview[]> {
+    const goalsDTO = await GoalService.getGoalsDTO();
+    const goalsPreview = goalsDTO.map((goalDTO) => (goalDTOToGoalPreviewConverter(goalDTO)));
```

Comment:
I think this syntax more readable, agreed? check everywherye
```suggestion
    const goalsPreview = goalsDTO.map(goalDTOToGoalPreviewConverter);
```

Reply:
Changed everywhere